src/_utils/ajax.js

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+
+const handleAxiosErrors = (error) => {
+  if (error.response) {
+    return error.response.data;
+  }
+
+  if (error.request) {
+    return error.request;
+  }
+
+  return error.message || 'Something went wrong! Please try again later';
+};
+
+const fetchInfo = (url, loadingCallback) => {
```

Comment:
Created this method in order to reuse it in other pages with the updated pagination logic. I'll update the other components once this is merged.